### DISCUSSION TITLE
To use asset local path on production

### DISCUSSION THREAD

**Discussion by clementroche** [tinacms/tinacms#4290] [Show and tell]
I noticed that tina uses `assets.tina.io` on production. I'm aware that's required for dealing with admin assets upload but i'm wondering if there is a clean way to use local path on prodction instead. Here is a quick snippet i'm adding on projects to do that so only admin uses `assets.tina.io` path

```js
function getTinaMedia(src) {
  const isBrowser = typeof window !== 'undefined'
  const isAdmin = isBrowser && window.location.pathname.includes('/admin')
  const isTina = src.includes('assets.tina.io')

  const localSrc = `/cms/${src.split('/').pop()}`

  return encodeURI(isAdmin || !isTina ? src : localSrc)
}
```

**Comment by jaschaio** [comment]
@clementroche where do you use that `getTinaMedia` method?
